modules/fb_instant_articles_rss/fb_instant_articles_rss.module

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Facebook Instant Articles RSS module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function fb_instant_articles_rss_menu() {
+  $items['fbinstant.rss'] = array(
+    'description' => 'Facebook Instant Articles RSS endpoint',
+    'page callback' => 'fb_instant_articles_rss_page',
+    'file' => 'includes/rss.inc',
+    'access arguments' => array('access content'),
+  );
+
+  $items['admin/config/services/fb-instant-articles/rss'] = array(
+    'title' => 'Facebook Instant Articles RSS',
+    'description' => 'RSS settings for Facebook Instant Articles.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('fb_instant_articles_rss_settings'),
+    'access arguments' => array('administer fb_instant_articles_rss'),
+    'file' => 'includes/admin.inc',
+  );
+
+  return $items;
+}
+
+/**
+ * Implements hook_permission().
+ */
+function fb_instant_articles_rss_permission() {
+  $permissions = array();
+
+  $permissions['administer fb_instant_articles_rss'] = array(
+    'title' => t('Administer Facebook Instant Articles RSS'),
+  );
+  return $permissions;
+}
+
+/**
+ * Implements hook_node_operations().
+ */
+function fb_instant_articles_rss_node_operations() {
+  // Check for setting "All nodes enabled by default?".
+  // Only show additional operations if setting is not set.
+  $fia_all_enabled = variable_get('fb_instant_articles_rss_all_enabled_default', TRUE);
+  if (!$fia_all_enabled) {
+    $operations = array(
+      'add_fia' => array(
+        'label' => t('Publish selected content to Facebook Instant Articles RSS feed'),
+        'callback' => 'fb_instant_articles_mass_update',
+        'callback arguments' => array('fia_enabled' => 1),
+      ),
+      'remove_fia' => array(
+        'label' => t('Unpublish selected content from Facebook Instant Articles RSS feed'),
+        'callback' => 'fb_instant_articles_rss_mass_update',
+        'callback arguments' => array('fia_enabled' => 0),
+      ),
+    );
+  }
+  return $operations;
```

Comment:
@vgardner I was going to leave things like this alone in this PR (if a bug already existed), but since you mentioned it here I just fixed this in 0a37d75